plugins/inputs/t128_graphql/config.go

```diff
@@ -4,73 +4,71 @@ import (
 	"strings"
 )
 
-//Config stores paths and paths to fields, tags and predicates used by queryBuilder and responseProcessor
+//Config stores paths to fields, tags and predicates to be used by BuildQuery and ProcessResponse
 type Config struct {
-	QueryPath  string
 	Predicates map[string]string
 	Fields     map[string]string
 	Tags       map[string]string
-	RawFields  map[string]string //Fields is used with RawFields to build the query
-	RawTags    map[string]string
 }
 
-//LoadConfig converts an entry point into a corresponding responsePath, queryPath and predicates
+/*
+LoadConfig converts a telegraf config into paths to predicates, fields and tags to be used by BuildQuery and ProcessResponse
+Paths correspond to keys in the output to avoid collisions and because they are used as lookups in ProcessResponse
+
+Args:
+	entryPoint example - "allRouters(name:'ComboEast')/nodes/nodes(name:'east-combo')/nodes/arp/nodes"
+	fieldsIn example - map[string]string{"test-field": "test-field"}
+	tagsIn example - map[string]string{"test-tag": "test-tag"}
+
+Example:
+	For the example input above, LoadConfig() will produce the following Config
+
+	*Config{
+		Predicates: map[string]string{
+			".data.allRouters.$predicate":             "(name:\"ComboEast\")",
```

Comment:
This is just internal, correct?

Reply:
It's internal. The user configures with a single quote